### DISCUSSION TITLE
Moving 2FA to a new phone

### DISCUSSION THREAD

**Discussion by wonder99** [community/community#102946] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have been using 2FA for many years, and am trying to configure it on my new phone.  I was able to generate the QR code for 2FA and read it in the Okta app on the new phone.  It recognizes it as my github account.  All looks good, but the codes are rejected upon login, and they're different that what's displayed on my old phone.  Is there a restriction that I can only have this on one phone?  Do I need to disable 2FA and then re-enable it for the new phone?

**Comment by DNin01** [comment]
This could be caused by a time desync. That is, your phone's clock does not match up with the clock at GitHub's servers. Since authenticator apps rely on the current time to display the correct one-time code, you'll need your clock to be set correctly. The best way to solve this is to check that your phone's time zone is correct and that your phone's clock is set/synchronized automatically.

**Comment by wonder99** [comment, marked as answer]
Thanks. I did confirm the time and date is correct on the new phone.
I avoided this problem by using a passkey instead of an 2fa app.

**Reply by DNin01** [reply]
That's a great solution! Glad it's working for you.
